heclib/heclib_f/src/zstags6.f

```diff
@@ -85,13 +85,4 @@ SUBROUTINE zstags6 (IFLTAB, CLINE, ISTAT)
       GO TO 800
 C
       END
-      SUBROUTINE zstags(IFLTAB, CLINE, ISTAT)
```

Comment:
Did you also check the source of DSSUTL, DSSMATH, DATCHK, etc..., or do we even build these anymore? I'm a little reluctant to remove a routine just because it's not called from within the library itself.

Reply:
We are not planning to build those tools with DSS7.  @MikeNeilson  is creating a new DSSUTL using C++.

And that routine, zstags,  would not work properly since it is passing a Real where a Character is expected.

Reply:
Well, that works for me.